plugins/Structural_Multipolygon.py

```diff
@@ -66,15 +66,15 @@ def relation(self, data, tags, members):
                 err_members.append(u"{0} - {1}".format(member['type'], member['role']))
 
         err = []
-        if len(err_roles) > 0:
-            err.append({"class": 11701, "subclass": 1, "text": {"en": ', '.join(err_roles)}})
         if len(err_members) > 0:
             err.append({"class": 11702, "subclass": 1, "text": {"en": ', '.join(err_members)}})
-
-        if outer == 0:
-            err.append({"class": 11703, "subclass": 1})
+        if len(err_roles) > 0:
+            err.append({"class": 11701, "subclass": 1, "text": {"en": ', '.join(err_roles)}})
         elif outer == 1 and inner == 0:
             err.append({"class": 11704, "subclass": 1})
+        if outer == 0:
+            err.append({"class": 11703, "subclass": 1})
+
```

Comment:
Please can you review all this changes, not required.

Reply:
Done!

I assume you agree with that warning 11704 should not be triggered if there's a typo in a role, e.g. a multipolygon with:
`outer`: a way
`inne`: a way inside the outer with a typo in the role

If not, I'll get rid of the `len(err_roles) == 0` check again

Reply:
thank, right.